Mathlib/RingTheory/Valuation/Basic.lean

```diff
@@ -236,18 +238,24 @@ theorem comap_comp {S₁ : Type*} {S₂ : Type*} [Ring S₁] [Ring S₂] (f : S
 
 /-- A `≤`-preserving group homomorphism `Γ₀ → Γ'₀` induces a map `Valuation R Γ₀ → Valuation R Γ'₀`.
 -/
-def map (f : Γ₀ →*₀ Γ'₀) (hf : Monotone f) (v : Valuation R Γ₀) : Valuation R Γ'₀ :=
```

Comment:
Are these changes supposed to be included?

Reply:
No indeed, I will fix when I get back to that PR (possibly in a while)